api/desecapi/models.py

```diff
@@ -502,7 +502,7 @@ def interval_label(self):
     'IPSECKEY',  # broken in pdns, https://github.com/PowerDNS/pdns/issues/10589 TODO enable with pdns auth > 4.5.0
     'KEY',  # Application use restricted by RFC 3445, DNSSEC use replaced by DNSKEY and handled automatically
     'WKS',  # General usage not recommended, "SHOULD NOT" be used in SMTP (RFC 1123)
-}
+} | {'NID', 'L32', 'L64', 'LP'}  # https://github.com/rthalley/dnspython/issues/674
```

Comment:
why use the union operator `|` here?

Reply:
Just for grouping, because I expect that we will remove the four of them together. I was too lazy "sorting them into the list" properly. (It would have required one line per entry, so should I have four comments, too?)

Reply:
I think it could be added as a single line in the first definition. But whatever :)